pkgs/development/rocm-modules/5/default.nix

```diff
@@ -318,6 +318,8 @@ in rec {
         rev = "640d7ee1917fcd3b6a5271aa6cf4576bccc7c5fb";
         sha256 = "sha256-T52whJ7nZi8jerJaZtYInC2YDN0QM+9tUDqiNr6IsNY=";
       };
+
+      patches = [ ./0001-Compile-transupp.c-as-part-of-the-library.patch ];
```

Comment:
We should add to oldAttrs.patches here just to be safe and don't overwrite any patches

Reply:
I am actually overwriting the old patches on purpose. I mention that in the commit message: https://github.com/NixOS/nixpkgs/pull/300257/commits/b3b12703803b2b0de008f291b245e7e5051ae6d5

Should I also mention it in a comment?

Reply:
I must admit I totally skipped the commit messages. Can you write that in one short comment like overwrite patches because they are for newer versions or so?

Reply:
i added a clarifying comment.